assets/locales/fr.json

```diff
@@ -584,7 +584,15 @@
   "security": {
     "security": "Sécurité",
     "gatekeeper": "Etat Gatekeeper",
-    "sip": "Etat SIP"
+    "gatekeeper_active": "Gatekeeper Active",
```

Comment:
Got it, I misunderstood a discussion on how to best do this on slack. I'll update to remove the added bits to both fr and de that I made.